data.json

```
@@ -1,55 +1,47 @@
 {
-    "WorkBench" : 
-        {
-            "SP-6" : 270,
-            "7N9 SPP" : 297,
-            "7N12 BP" : 320,
-            "Gunpowder-Eagle" : 98,
-            "Gunpowder-Hawk" : 36
-        },
+    "WorkBench": {
+        "SP-6" : 270,
```

```suggestion
        "SP-6": 270,
```

`:` 쓸때는 왼쪽을 붙이고 오른쪽을 띄워둡니다! 사실 이건 프로그래밍 언어 뿐만 아니라 그냥 문법 자체가 이렇게 되어있을걸요?

우리나라에서도 사과: 100원, 바나나: 200원 이렇게 `:` 쓸때는 왼쪽 붙이는게 맞춤법에 맞아요.